<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>991</width>
      <height>691</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>Início</string>
     </attribute>
     <widget class="QPushButton" name="btn_carregar">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>20</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Carregar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="file_field">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>641</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>C:\Users\pedro\Documents\Projetos\git\Inventario_ACV\entradas.xlsx</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QToolButton" name="btn_procurar">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>20</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset theme="open.xmp"/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_calc_inv">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>540</y>
        <width>761</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Calcular inventário</string>
      </property>
     </widget>
     <widget class="QTableView" name="tvB">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>100</y>
        <width>241</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>60</y>
        <width>21</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Sans Serif</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditnprocess">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>480</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Sans Serif</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>60</y>
        <width>21</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Sans Serif</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>K</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>21</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Sans Serif</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>470</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Sans Serif</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Número de produtos</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>470</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Sans Serif</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Número de processos</string>
      </property>
     </widget>
     <widget class="QTableView" name="tvA">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>241</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditnprod">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>480</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Sans Serif</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTableView" name="tvk">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>100</y>
        <width>241</width>
        <height>311</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>Processar A</string>
     </attribute>
     <widget class="QGroupBox" name="group_metodo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>841</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Método para tornar A quadrada</string>
      </property>
      <widget class="QRadioButton" name="fantasma">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Processo Fantasma</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="corte">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>60</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Critério de corte</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="alocacao">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>30</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Alocação</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>100</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Produto</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_prod_criterio">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>110</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_proc_aloc">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>80</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>70</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Processo</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_prod1_aloc">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>120</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>110</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Produto 1</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_prod2_aloc">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>160</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>150</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Produto 2</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btn_calcular_matrizes">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>240</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Calcular matrizes</string>
      </property>
     </widget>
     <widget class="QTableView" name="tvB2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>330</y>
        <width>241</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>280</y>
        <width>21</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Sans Serif</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>280</y>
        <width>21</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Sans Serif</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>K</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_calc_inventario">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>610</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Calcular inventário</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>280</y>
        <width>21</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Sans Serif</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QTableView" name="tvk2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>330</y>
        <width>241</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tvA2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>330</y>
        <width>241</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Resultados</string>
     </attribute>
     <widget class="QTableView" name="tvbinva">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>70</y>
        <width>241</width>
        <height>561</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>20</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Sans Serif</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>B*invA</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>20</y>
        <width>21</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Sans Serif</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>M</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Sans Serif</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>InvA</string>
      </property>
     </widget>
     <widget class="QTableView" name="tvm">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>70</y>
        <width>341</width>
        <height>561</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tvinva">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>241</width>
        <height>561</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
